README.md

```diff
@@ -11,52 +11,43 @@ Travis
 [![Build
 Status](https://travis-ci.org/openMF/mifosx.png?branch=master)](https://travis-ci.org/openMF/mifosx)
 
-Cloudbees Jenkins
-
-[![Build
-Status](https://openmf.ci.cloudbees.com/job/MIFOSX%20INTEGRATION%20TEST/badge/icon)](https://openmf.ci.cloudbees.com/job/MIFOSX%20INTEGRATION%20TEST/)
-
-<a target="_blank" href="https://openmf.ci.cloudbees.com/job/MIFOSX%20INTEGRATION%20TEST/"  title="Jenkins@CloudBees">Jenkins@CloudBees Unit + Integration Tests</a>
-
 
 Version
 ==========
 
-The API for the mifos-platform (project named 'Mifos X')is documented in the api-docs under <b>Full API Matrix</b> and can be viewed <a target="_blank" href="https://demo.openmf.org/api-docs/apiLive.htm" title="API Documentation"> here
-</a>
+The API for the Mifos-platform (project named 'Mifos X') is documented in the API-docs under <b>Full API Matrix</b> and can be viewed <a href="https://demo.openmf.org/api-docs/apiLive.htm" title="API Documentation"> here</a>.
 
-Latest stable release can always been viewed on master branch: <a target="_blank" href="https://github.com/openMF/mifosx/tree/master" title="Latest Release">Latest Release on Master</a>, <a target="_blank" href="https://github.com/openMF/mifosx/blob/master/CHANGELOG.md" title="Latest release change log">View change log</a>
+The latest stable release can be viewed on the master branch: <a href="https://github.com/openMF/mifosx/tree/master" title="Latest Release">Latest Release on Master</a>, <a href="https://github.com/openMF/mifosx/blob/master/CHANGELOG.md" title="Latest release change log">View change log</a>
 
 License
 =============
 
-This project is licensed under the open source MPL V2. See https://github.com/openMF/mifosx/blob/master/LICENSE.md
+This project is licensed under open source MPL V2. See <https://github.com/openMF/mifosx/blob/master/LICENSE.md>.
 
-Mifos Platform API
+Mifos X Platform API
 =====================
 
-<a target="_blank" href="https://demo.openmf.org/api-docs/apiLive.htm" title="mifos platform api">API Documentation (Demo Server)</a>
```

Comment:
target="_blank" is not supported in markdown. It will open up in the same page. Do you still want me to keep it?